src/value.cpp

```diff
@@ -495,18 +496,19 @@ namespace Exiv2 {
         std::string c = value_;
         if (charsetId() == unicode) {
             c = value_.substr(8);
-            std::string::size_type sz = c.size();
+            const size_t sz = c.size();
             if (byteOrder_ == littleEndian && byteOrder == bigEndian) {
                 convertStringCharset(c, "UCS-2LE", "UCS-2BE");
```

Comment:
Maybe we should check the return value of this function? It's just a wrapper around iconv on Linux (and not implemented on Windows).

Reply:
I will investigate take a deeper look and create an issue to not forget about it in the future. In this PR I just want to focus on the warnings :)